scripts/cleanup-dcp1/readme.md

```diff
@@ -0,0 +1,11 @@
+# Cleanup DCP1 projects
+This script cleans any the terra staging area of DCP1 projects so that they are ready to export again if changes have been made
+
+**You need to have gsutil setup with the correct credentials prior to running this**
```

Comment:
could link to the SOP in setting up terra access

Reply:
please put the link here as I have to do this tomorrow.

Reply:
https://ebi-ait.github.io/hca-ebi-dev-team/admin_setup/Setting-up-access-to-Terra-staging-area.html